### PR TITLE
Remove Tailwind References from App

### PR DESCRIPTION
**Pull request by jbouder**
## Reference Issues or PRs

N/A

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [x] Did you add new tests?

## Screenshots
Screenshot 2024-03-07 at 12 46 58 PM
Screenshot 2024-03-07 at 12 47 11 PM
Screenshot 2024-03-12 at 11 35 45 AM
Screenshot 2024-03-07 at 12 56 02 PM

## Any other comments?
- Replaced all applicable custom components with MUI components
- Replaced all use of tailwind utility classes with MUI components and custom classes
- Updates to ensure consistent theming
- Removed all tailwind dependencies
- Misc UI updates to better support use of MUI components

### CODE REVIEW

File: jhub_apps/service/japps_routes.py [hunk 1]
```diff
@@ -14,4 +16,8 @@
 @router.get("/edit-app", response_class=HTMLResponse)
 @router.get("/server-types", response_class=HTMLResponse)
 async def handle_apps(request: Request):
-    return templates.TemplateResponse("japps_home.html", {"request": request})
+    now = datetime.now()
```
Comment (aktech) [inline jhub_apps/service/japps_routes.py:19]: I think its worth passing in `timezone.utc` for consistency and reproducibility.

```python
from datetime import timezone
datetime.now(timezone.utc)
```

**Comment by aktech** [comment]
I tried this locally, everything looks fine. This is good to merge, I have a minor comment.

File: jhub_apps/service/japps_routes.py (hunk 1 above)
Comment (jbouder) [reply jhub_apps/service/japps_routes.py]: Fixed

**Comment by aktech** [comment]
@jbouder Quick question, we'll be styling this later?

Screenshot 2024-03-12 at 4 16 19 pm


---
Earlier it looked something like this:
Screenshot 2024-03-12 at 4 16 40 pm

**Comment by jbouder** [comment]
It was changed to a text, secondary styling based on recent feedback, which is what you see now. There are some inconsistencies in positioning and spacing between screens (forms, modals, etc), but this matches the current guidance

**Comment by aktech** [comment]
Thanks for clarification!

**Pull request merged**